docs/abstractions.py

```diff
@@ -120,7 +120,8 @@ class LoadOps(Enum):     EMPTY = auto(); RAND = auto(); CONST = auto(); FROM = a
 from tinygrad.ops import LazyOp, BinaryOps, LoadOps
 
 # the 2+3 from before
-result = Tensor([2]) + Tensor([3])
+# added some 0s, otherwise Tensor([2]) will be folded into a constant without using LoadOps.FROM
+result = Tensor([2, 0]) + Tensor([3, 0])
```

Comment:
Can you test if adding `.realize()` instead of the 0s will force it into a buffer?

Reply:
This part of the doc inspects the ops in lazydata before realizing, so we should not realize it early